appveyor.yml

```diff
@@ -7,7 +7,8 @@ image: Visual Studio 2017
 nuget:
   project_feed: true
   disable_publish_on_pr: true
-  
+version: 1.9.2.{build}
```

Comment:
I specifically removed this because it will cause duplicate build numbers. Besides, it will have no effect on the outcome because the build number is then changed programmatically using the `Update-AppveyorBuild` command. It doesn't hurt either but it feels redundant to have this and `ProductVersion` variable below